pipeline/release/release.go

```diff
@@ -67,9 +73,13 @@ func description(diff string) string {
 	return result + "\nBuilt with " + string(bts)
 }
 
-func upload(client *github.Client, releaseID int, archive string, ctx *context.Context) error {
-	archive = archive + "." + ctx.Config.Archive.Format
-	file, err := os.Open("dist/" + archive)
+func upload(ctx *context.Context, client *github.Client, releaseID int, archive, format string) error {
+	archive = archive + "." + format
+	var path = filepath.Join("dist", archive)
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return nil
```

Comment:
Yeah, I added this part because the current code will try to add the mac build `.deb` too..

In this case I'll just ignore it... If the archive wasn't generated and should, it should have failed in the archive or fpm pipe I think..

Maybe it would be better to add a list of uploads in the context...